developers/blobstream.md

```diff
@@ -14,89 +14,111 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
+:::tip NOTE
+Custom ranges can be requested using the `BlobstreamX` contract
+to create proofs for specific Celestia block batches. These ranges
+can be constructed as `[latestBlock, customTargetBlock)`, with
+`latestBlock` is the latest block height that was committed to by the
+`BlobstreamX` contract, and `latestBlock > customTargetBlock`,
+and `customTargetBlock - latestBlock <= DATA_COMMITMENT_MAX`.
+
+Block ranges that are before the contract's `latestBlock` can't be
+proven a second time in different batches.
+
+More information can be found in [`requestHeaderRange(...)`](https://github.com/succinctlabs/blobstreamx/blob/364d3dc8c8dc9fd44b6f9f049cfb18479e56cec4/contracts/src/BlobstreamX.sol#L78-L101)
+method.
+:::
 
-### Events and messages relayed
+### How Blobstream X works
 
-**Validator sets**:
-The relayer informs the Blobstream contract who are the current
-validators and their power.
-This results in an execution of the `updateValidatorSet` function.
+As shown in the diagram below, the entrypoint for updates to the Blobstream
+X contract is through the `SuccinctGateway` smart contract, which is a
+simple entrypoint contract that verifies proofs (against a deployed
+onchain verifier for the Blobstream X circuit) and then calls the
+`BlobstreamX.sol` contract to update it. More information about the
+`SuccinctGateway` can be found here: [TODO].
 
-**Batches**:
-The relayer informs the Blobstream contract of new data root tuple roots.
-This results in an execution of the `submitDataRootTupleRoot` function.
+![blobstream x overview diagram draft](/img/blobstream/blobstream_x_overview_diagram_draft.png)
 
-## How to integrate
+<!-- markdownlint-disable MD042 -->
 
-Integrating your L2 with Blobstream requires two components: your onchain smart
+:::tip NOTE
+If the contract is still not deployed, then it needs to be
+deployed before it is used by the prover/relayer. See the
+[deployment documentation (TBD)]() for more details.
+:::
+
+### How to integrate with Blobstream X
+
+Integrating your L2 with Blobstream X requires two components: your onchain smart
 contract logic, and your offchain client logic. The next three sections cover these
 topics:
 
-- [Integrate with Blobstream contracts](./blobstream-contracts.md)
-- [Integrate with Blobstream client](./blobstream-offchain.md)
-- [Querying the Blobstream proofs](./blobstream-proof-queries.md)
+- [Integrate with Blobstream X contracts](./blobstreamx-contracts.md)
+- [Integrate with Blobstream X client](./blobstreamx-offchain.md)
+- [Querying the Blobstream X proofs](./blobstreamx-proof-queries.md)
 
 ### Deployed contracts
 
-You can interact with the Blobstream contracts today on testnet. The Blobstream Solidity
-smart contracts are currently deployed on the following Ethereum testnets:
+You can interact with the Blobstream X contracts today on testnet. The
+Blobstream X Solidity smart contracts are currently deployed on
```

Comment:
Should we change these to all just be deployed Blobstream X instances?

Reply:
yes, can you please link them here (once deployed)?

Reply:
I will modify this PR today to:

- [ ] create a section with networks (Arbitrum One, Base, ETH) that we'll launch on, with placeholders for contract addresses

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/2c6a969a453e10e96e9ed3b1d956a80de8018a4c

Reply:
this is still open to:

- [ ] modify the contract addresses once they have been deployed

Reply:
resolving to bring this comment out